lib/backend/postgres/index.ts

```diff
@@ -398,22 +407,24 @@ export class PostgresBackend implements Database {
 
 	/*
 	 * This method first ensures that the migrations table exists, and then
-	 * executes the provided callback in a transaction if the installed version
-	 * of autumndb is newer than what is in the migrations table.
+	 * executes the provided callback in a transaction if the provided version
+	 * is newer than what is in the migrations table.
```

Comment:
I'd add a reference to the `name` of the migration. Something like
"if the provided version of the migration named `name`"

Reply:
Thanks for the reminder, `name` and `requires` are new arguments and I've forgotten to update the comment.

Reply:
Done, updated docs for this method